internal/service/alarms/internal/db/migrations/000003_create_alarm_event_record_table.up.sql

```diff
@@ -57,3 +56,40 @@ CREATE OR REPLACE TRIGGER update_alarm_event_sequence
     BEFORE UPDATE ON alarm_event_record
     FOR EACH ROW
     EXECUTE FUNCTION update_alarm_event_sequence();
+
+-- Function to track alarm_changed_time
+CREATE OR REPLACE FUNCTION track_alarm_event_record_change_time()
```

Comment:
As far as I know the triggers are executed in alphabetical order, so this works fine. But I usually see that a single function is created where these functions are called inside. To not depend on that. Perhaps, later

Reply:
yeah that's true. So in this case we would like `track_alarm_event_record_change_time` to execute before `update_alarm_event_sequence`. To make it prod friendly we would need to do something `CREATE TRIGGER ... AFTER TRIGGER another_trigger_name`. Will this change later;. Keeping it unresolved till then.